### PR TITLE
Remove Serverless-Now from hub-stack. Added hotfix for subhub.spec to allow running of tests.

### PR DESCRIPTION
**Pull request by warrickct**
* Removed serverless now from the hubstack monorepo as it's independent of the hub stack now. Removed references to it in the Jenkins pipeline.
* Not related to the removal but I also added a conversion to 'unknown' data type in the mock data types for subhubs to stop generating errors due to schema generation type and mock data type mismatch.

### CODE REVIEW

**Comment by warrickct** [comment]
* Deployment on Jenkins working successfully 6c55486607338d77e78b4640cbb4112bb8b39559.
* Cer-graphql tests passing locally.
* Research-hub-web e2e and unit tests passing locally.

File: research-hub-web/src/app/components/subhubs/subhubs.component.spec.ts [hunk 1]
```diff
@@ -143,7 +143,7 @@ describe('SubhubsComponent', () => {
       }
     ],
     "__typename": "SubHubCollection"
-  } as SubHubCollection);
+  } as unknown as SubHubCollection);
```
Comment (JuliusKarl) [inline research-hub-web/src/app/components/subhubs/subhubs.component.spec.ts:146]: I also had to add `as unknown` to bypass some overlap issues but I still don't know why

**Comment by JuliusKarl** [comment]
Looks good, Can you merge master and see that it works with all the new updates.

**Review by JuliusKarl — COMMENTED** [review]
Looks good, can you merge with master and see that it works with the new updates (i.e. _terraform_ and _graphql-updates_)

**Pull request merged**